### PR TITLE
[REF] mail: `RtcLayoutMenu`, review and simplify scss

### PR DESCRIPTION
**Pull request by Brieuc-brd**
Part of the overall v16 SCSS optimization/restyle, task-2704984

task-2855553

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Brieuc-brd** [comment]
Hello @alexkuhn 👋
This PR is ready for review.

cc. @stefanorigano

File: addons/mail/static/src/components/rtc_layout_menu/rtc_layout_menu.scss [hunk 1]
```diff
@@ -2,45 +2,16 @@
...
-
-.o_RtcLayoutMenu_radio {
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
-    font-size: 1.3em;
-}
-
 .o_RtcLayoutMenu_radioIcon {
     width: 5rem;
```
Comment (stefanorigano) [inline addons/mail/static/src/components/rtc_layout_menu/rtc_layout_menu.scss:6]: The image dimensions can be set directly on the SVG code (unless these must be somehow dynamic, but I don't think it's the case)
https://css-tricks.com/scale-svg/

File: addons/mail/static/src/components/rtc_layout_menu/rtc_layout_menu.scss [hunk 2]
```diff
@@ -2,45 +2,16 @@
...
 // ------------------------------------------------------------------
 // Style
 // ------------------------------------------------------------------
 
 .o_RtcLayoutMenu_radio {
-    cursor: pointer;
-    border-radius: 5px;
-
     &:hover {
         background-color: gray('100');
```
Comment (stefanorigano) [inline addons/mail/static/src/components/rtc_layout_menu/rtc_layout_menu.scss:16]: Not a big fun of these custom fields design :/

For the scope of this task we could use default bootstrap components, https://getbootstrap.com/docs/4.3/components/forms/#radios.

If the visual result changes too much compared  to master, then let's use a combination of utility classes (bg-X, opacity-X-hover..)

**Review by stefanorigano — COMMENTED** [review]
Hello @Brieuc-brd , some remarks

File: addons/mail/static/src/components/rtc_layout_menu/rtc_layout_menu.scss (hunk 1 above)
Comment (Brieuc-brd) [reply addons/mail/static/src/components/rtc_layout_menu/rtc_layout_menu.scss]: Updated 👍

**Review by alexkuhn — CHANGES_REQUESTED** [review]
Would be nice to make a different commit for indentation cleaning.

There are some other tasks that make changes to `rtc_layout_menu.xml`, and it would be much easier to rebase and solve conflicts when it's not mixed together.

Thanks!

**Comment by Brieuc-brd** [comment]
> Would be nice to make a different commit for indentation cleaning.
> 
> There are some other tasks that make changes to `rtc_layout_menu.xml`, and it would be much easier to rebase and solve conflicts when it's not mixed together.
> 
> Thanks!

I created a new commit for the indentation cleaning 👍 
What do you think ?

**Review by alexkuhn — APPROVED** [review]
This is better!

I can rebase and solve conflicts in-between, which is a much better experience.
With just a single commit, the lesser pain is either re-do these changes or my changes by hand.

Thanks!

@robodoo r+ rebase-ff

**Pull request closed without merging**